locales/en-US/messages.properties

```diff
@@ -86,6 +86,7 @@ Bitcoin=Bitcoin
 donation_notice_bitcoin=The Mozilla Foundation is a California non-profit corporation exempt from United States federal income taxation under IRC 501(c)(3) and a public charity classified under IRC sections 170(b)(1)(A) and 509(a)(1). Bitcoin donations Mozilla receives are considered charitable contributions under U.S. federal tax laws, to be used in its discretion for its charitable purposes. If you have questions, check out <a href="https://wiki.mozilla.org/Donate">our FAQ</a> or contact us at <a href="mailto:donate@mozilla.org">donate@mozilla.org</a>.
 email_invalid=Please make sure you have entered a valid email address.
 firefox_footer=Mozilla is the non-profit organization behind Firefox.
+donate_to_mozilla=Donate to Mozilla
```

Comment:
What's the process like for this if it's not ready yet in the 5 priority languages?

I only ask because we wanted to ship 25% snippet traffic to our 5 target locales tomorrow.